core/src/main/java/com/aem/demo/core/models/NewsCardModel.java

```diff
@@ -45,12 +46,9 @@ public class NewsCardModel {
     @Default(values = DEFAULT_IMAGE)
     private String image;
 
-    private Integer likes = 0;
+    private Integer likes;
```

Comment:
Please mind that now, as you moved all the "incrementing" logic away from this class, you would rather go with injecting values with ordinary `@ValueMapValue`, etc. than reading them in `PostConstruct`. Because that's how you handle, for example, `image` etc,
Also please mind changing Boxed numbers back to primitives because this is what's expected of an ordinary class field.

See: code is like an organism. It's not like a... pile of stones) Think of code as a plant, some tree. When you do something to one part, it may affect other parts. And if you don't care, your plant goes hurt or starts looking half-broken) 

Once there was an incrementation logic. Lines like `likes = valueMap.get("like", Integer.class);` were part of that logic. Now you removed that logic. And the first thing you oughtta do next, is: stop and think: "How's the rest of the code is looking now? Isn't it looking like a crooked tree bent to one side because I have just cut off branches from the opposite side? What should I do to make it stand straight again?)"

This is important. This is what distinguishes a true programmer -- ability to thing of a whole picture, bear the whole picture in one's mind.

You do change things that you are asked to change, and usually, you do this quite accurately.
But then you go forget about *neighboring* things! For example, you can change a line here because you were asked to, but then you walk away, not even looking back, and not noticing the very similar line which is some 10 lines below (because you were not asked to).

This exactly sometimes produces the feeling that you are doing things in a great hurry) However we do know that isn't true,.